help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,161 @@
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+> 
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/salesforce/catalog.png)
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![Account Info](../../assets/catalog/crm/salesforce/account-info.png)
+
+*  **[!UICONTROL Password]**: Your Salesforce account password.
+*  **[!UICONTROL Client ID]**: Your Salesforce connected app Consumer Key.
+*  **[!UICONTROL Client Secret]**: Your Salesforce connected app Consumer Secret.
+*  **[!UICONTROL Username]**: Your Salesforce account username.
+
+If the details provided are valid, the UI displays a **Connected** status with a green check mark, you can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+![Destination Details](../../assets/catalog/crm/salesforce/destination-details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Custom Domain]**: Your Salesforce domain.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+## Mapping Example {#mapping-example}
+
+In the select target window, When selecting the target field choose the **Select custom attributes** category and add the mappings' desired.
+
+For instance, your XDM profile schema and your [!DNL Salesforce CRM] instance could have the attribute mapping as below:
```

Comment:
Minor rewording suggestion: 

For instance, you could add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance: